modules/components/AsyncConnect.js

```diff
@@ -43,10 +44,6 @@ export default class AsyncConnect extends Component {
     }
   }
 
-  componentWillReceiveProps(nextProps) {
```

Comment:
Hmm.
The problem I found was that with calling `this.loadAsyncData` in `componentWillReceiveProps` _and_ `componentDidMount`, it would make two requests, and dispatch BEGIN_GLOBAL_LOAD, END_GLOBAL_LOAD twice.

Do you have a suggestion on how to make it smarter so it doesn't request twice?